src/odin/enhancedtrippath.cc

```diff
@@ -483,7 +483,7 @@ std::string EnhancedTripPath_Edge::ToParameterString() const {
 
   str += delim;
   str += "TripPath_RoadClass_";
-  str += TripPath_RoadClass_descriptor()->FindValueByNumber(road_class())->name();
```

Comment:
Does this return the name like before?

Reply:
no its just a number, the number of the enum. if we want i can add all the functions. i figured for something that is only on for `LOG_TRACE` this was ok. let me know if this is unaccaptable and i can do the extra work to call the right functions.

speaking of this. one of the converage things we are getting dinged on is all these lines of code that are only enabled for `LOG_TRACE`. Shouldn't we we put `#ifdef`s around these functions so that the coverage code doesnt even see them?

Reply:
it was nice to have the names while using trace - but for time sake we can skip for now. if it makes sense - then yeah we could add those ifdef